assets/xml/objects/object_sdn.xml

```diff
@@ -1,66 +1,66 @@
-        <Texture Name="object_sdn_Tex_00C400" OutName="tex_00C400" Format="ci8" Width="32" Height="32" Offset="0xC400" />
-        <Texture Name="object_sdn_Tex_00C800" OutName="tex_00C800" Format="i8" Width="16" Height="16" Offset="0xC800" />
-        <Texture Name="object_sdn_Tex_00C900" OutName="tex_00C900" Format="i8" Width="32" Height="32" Offset="0xC900" />
-        <Texture Name="object_sdn_Tex_00CD00" OutName="tex_00CD00" Format="ci8" Width="8" Height="8" Offset="0xCD00" />
-        <Texture Name="object_sdn_Tex_00CD40" OutName="tex_00CD40" Format="ci8" Width="16" Height="16" Offset="0xCD40" />
-        <Texture Name="object_sdn_Tex_00CE40" OutName="tex_00CE40" Format="ci8" Width="8" Height="8" Offset="0xCE40" />
-        <Texture Name="object_sdn_Tex_00CE80" OutName="tex_00CE80" Format="ci8" Width="8" Height="8" Offset="0xCE80" />
-        <Texture Name="object_sdn_Tex_00CEC0" OutName="tex_00CEC0" Format="ci8" Width="16" Height="16" Offset="0xCEC0" />
-        <Texture Name="object_sdn_Tex_00CFC0" OutName="tex_00CFC0" Format="ci8" Width="32" Height="32" Offset="0xCFC0" />
-        <Texture Name="object_sdn_Tex_00D3C0" OutName="tex_00D3C0" Format="i8" Width="16" Height="8" Offset="0xD3C0" />
-        <Texture Name="object_sdn_Tex_00D440" OutName="tex_00D440" Format="ci8" Width="16" Height="16" Offset="0xD440" />
-        <Limb Name="object_sdn_Standardlimb_00D540" Type="Standard" Offset="0xD540" />
-        <Limb Name="object_sdn_Standardlimb_00D54C" Type="Standard" Offset="0xD54C" />
-        <Limb Name="object_sdn_Standardlimb_00D558" Type="Standard" Offset="0xD558" />
-        <Limb Name="object_sdn_Standardlimb_00D564" Type="Standard" Offset="0xD564" />
-        <Limb Name="object_sdn_Standardlimb_00D570" Type="Standard" Offset="0xD570" />
-        <Limb Name="object_sdn_Standardlimb_00D57C" Type="Standard" Offset="0xD57C" />
-        <Limb Name="object_sdn_Standardlimb_00D588" Type="Standard" Offset="0xD588" />
-        <Limb Name="object_sdn_Standardlimb_00D594" Type="Standard" Offset="0xD594" />
-        <Limb Name="object_sdn_Standardlimb_00D5A0" Type="Standard" Offset="0xD5A0" />
-        <Limb Name="object_sdn_Standardlimb_00D5AC" Type="Standard" Offset="0xD5AC" />
-        <Limb Name="object_sdn_Standardlimb_00D5B8" Type="Standard" Offset="0xD5B8" />
-        <Limb Name="object_sdn_Standardlimb_00D5C4" Type="Standard" Offset="0xD5C4" />
-        <Limb Name="object_sdn_Standardlimb_00D5D0" Type="Standard" Offset="0xD5D0" />
-        <Limb Name="object_sdn_Standardlimb_00D5DC" Type="Standard" Offset="0xD5DC" />
-        <Limb Name="object_sdn_Standardlimb_00D5E8" Type="Standard" Offset="0xD5E8" />
-        <Limb Name="object_sdn_Standardlimb_00D5F4" Type="Standard" Offset="0xD5F4" />
-        <Skeleton Name="object_sdn_Skel_00D640" Type="Flex" LimbType="Standard" Offset="0xD640" />
-        <Animation Name="object_sdn_Anim_00DC7C" Offset="0xDC7C" />
+        <DList Name="gDemoheishiBottleDL" Offset="0x630" />
```

Comment:
```suggestion
        <DList Name="gGuardBottleDL" Offset="0x630" />
```
There are multiple actors that share this object so it doesn't really make sense to name it after just one of them. I'd probably lean toward guard for the prefix but if anyone else has any ideas feel free to share.

Reply:
I went ahead with gGuardBottleDL in the meantime, it seems to fit.

Reply:
Oh I meant for this to apply to everything in `object_sdn.xml` not just the `Bottle DLs`.

Reply:
Ah, that makes more sense to me now. I updated the properties so that now they refer to Soldier as opposed to Demoheishi specifically,